src/lib/CollisionPrevention/CollisionPrevention.hpp

```diff
@@ -84,21 +83,12 @@ class CollisionPrevention : public ModuleParams
 
 	hrt_abstime _last_message;
 
-	matrix::Vector2f _move_constraints_x_normalized;
-	matrix::Vector2f _move_constraints_y_normalized;
-	matrix::Vector2f _move_constraints_x;
-	matrix::Vector2f _move_constraints_y;
-
 	DEFINE_PARAMETERS(
 		(ParamFloat<px4::params::MPC_COL_PREV_D>) _param_mpc_col_prev_d /**< collision prevention keep minimum distance */
 	)
 
 	void update();
 
-	void update_range_constraints();
-
-	void reset_constraints();
-
-	void publish_constraints(const matrix::Vector2f &original_setpoint, const matrix::Vector2f &adapted_setpoint);
+	void calculate_constrained_setpoint(matrix::Vector2f &setpoint, const float max_acc);
```

Comment:
```suggestion
	void calculateConstrainedSetpoint(matrix::Vector2f &setpoint, const float max_acc);
```
Try to use `camelCase` to name the functions and keep `snake_case` for the variables